src/interfaces/tokens/IProtocolToken.sol

```diff
@@ -4,9 +4,25 @@ pragma solidity 0.8.20;
 import {IERC20Metadata} from '@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol';
 import {IERC20Permit} from '@openzeppelin/contracts/token/ERC20/extensions/IERC20Permit.sol';
 import {IVotes} from '@openzeppelin/contracts/governance/utils/IVotes.sol';
+
 import {IAuthorizable} from '@interfaces/utils/IAuthorizable.sol';
 
 interface IProtocolToken is IERC20Metadata, IERC20Permit, IVotes, IAuthorizable {
+  // --- Errors ---
+
+  /// @notice Throws when trying to pause the token a second time
+  error ProtocolToken_NotPausable();
+
+  // --- Data ---
+
+  /**
+   * @notice Pausability status of the token
+   * @return _notPausable Whether the token is pausable or not
+   */
+  function notPausable() external view returns (bool _notPausable);
```

Comment:
whhattttt? why using `pausable`?

Reply:
It was used to introduce a pausability status, in addition to the pause state. However, as you pointed out, it is not the best approach here as we know upfront when it should no longer be pausable.